module/data/item/spell.mjs

```diff
@@ -190,6 +203,30 @@ export default class SpellData extends ItemDataModel.mixin(ActivitiesTemplate, I
     }
   }
 
+  /* -------------------------------------------- */
+
+  /**
+   * Migrate preparation data.
+   * @since 5.1
```

Comment:
Linter wants this to be an actual SemVer version. So we could adjust it or disable the `jsdoc/check-values` rule (which is only used for a handful of tags anyway: https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/check-values.md)

```suggestion
   * @since 5.1.0
```